src/Create.js

```diff
@@ -0,0 +1,130 @@
+        console.log(data);
+        if (data.error) {
+            this.setState({ message: data.error, error: true });
+        } else {
+          this.props.history.push('/');
+        }
+    }
+
+    render() {
+        return (
+            <>
+             {this.state.message && (
+                    <div
+                        className={classNames({
+                            message: true,
+                            error: this.state.error,
+                            success: !this.state.error,
+                        })}
+                    >
+                        {this.state.message}
+                    </div>
+                )}
+                <h1>{this.state.name}</h1>
+                <img className='create-image' src={this.state.image_url} width='500' alt='game' />
+                <form id='create-videogame'>
+                <div className="forms">
+                    <label>Name: </label>
+                    <input onChange={(e) => {
+                        this.setState({ name: e.target.value});
+                    }}
+                    type='text'
+                    value={this.state.name}>
+                    </input>
+                </div>
+                <div className="forms">
+                    <label>Image(URl): </label>
+                    <input className='image-input' onChange={(e) => {
+                        this.setState({ image_url: e.target.value });
+                    }}
+                    type='text' value={this.state.image_url}>
+                    </input>
+                </div>
+                <div className="forms">
+                    <label>System</label>
+                    <select value={this.state.system}
+                        onChange={(e) => {
+                        this.setState({ system: e.target.value });}}>
+                        {this.state.systems.map((item) => {
+                        return (<option value={item.console}>{item.console}</option>
+                            );
+                    })}
+                    </select>
+                </div>
+                <div className="forms">
+                        <label>Year Released: </label>
+                        <input onChange={(e) => {
+                            this.setState({
+                                year_released: e.target.value
+                            });
+                        }}
+                        type='number'
+                        value={this.state.year_released}></input>
+                </div>
+                <div className="forms">
+                        <label>Have you played? </label>
+                        <input name='radio-btn' onChange={(e) => {
+                            this.setState({
+                                played: e.target.value
+                            });
+                        }}
+                        type='radio'
+                        value={true}></input>
```

Comment:
Since you aren't actually placing anything inside of this `<input>` tag, you can actually just use the self-closing syntax: `<input />`